tcms/testruns/static/testruns/js/get.js

```diff
@@ -282,6 +294,35 @@ function renderTestExecutionRow(testExecution) {
     template.find('.test-execution-status-icon').addClass(testExecutionStatus.icon).css('color', testExecutionStatus.color)
     template.find('.test-execution-status-name').html(testExecutionStatus.name).css('color', testExecutionStatus.color)
 
+    const commentField = template.find('.comment-text')[0]
+    const fileInputField = template.find('.simplemde-file-upload')
```

Comment:
The textarea and upload fields need unique IDs:
https://github.com/kiwitcms/Kiwi/pull/2029/commits/cce2ac682668cc2b5241bcc43a1625ca1a7bb610